check_warning_limit.py

```diff
@@ -0,0 +1,63 @@
+from Print import print_warning,German_pring_warning
+
+def warning_function(value,min,max,function_name):
+    warning_value = Warning_value_finder(min, max)
+    feedback = warning_check_function(value,min,warning_value[0],warning_value[1],max)
+    if(feedback == True):
+        print_warning(function_name)
+        German_pring_warning(function_name)
```

Comment:
Both languages supported are used for printing warning messages. Generally based on configuration only English or German strings should be printed.
Please think on how to pick language strings based on configuration. Maybe you can store strings in arrays and use relevant language strings based on config!